### PR TITLE
fix: transactions state

### PR DESCRIPTION
**Pull request by alex-sandrk**
Related: https://github.com/Lumerin-protocol/WalletDesktop/pull/8

### CODE REVIEW

File: src/reducers/wallet.js [hunk 1]
```diff
@@ -10,18 +10,22 @@ export const initialState = {
...
+const mergeTransactions = (stateTxs, payloadTxs) => {
+  return [...stateTxs, ...payloadTxs].filter(
+    tx =>
+      tx.receipt ||
+      (!tx.receipt &&
+        !payloadTxs.find(
+          t => t.receipt && t.transaction.hash === tx.transaction.hash
+        ))
+  )
+}
```
Comment (lsheva) [inline src/reducers/wallet.js:27]: unclear intention of this function
extract variable or leave a comment

File: src/utils/createTransactionParser.js [hunk 2]
```diff
@@ -139,11 +133,12 @@ export const createTransactionParser = myAddress => rawTx => {
     isProcessing: getIsProcessing(tokenData),
     blockNumber: getBlockNumber(rawTx),
     isApproval: getIsApproval(tokenData),
+    isPending: getIsPending(rawTx),
     timestamp,
     gasUsed: getGasUsed(rawTx),
     txType,
     symbol: getSymbol(tokenData, txType),
-    value: getValue(rawTx, tokenData, txType),
+    value: getValue(rawTx, tokenData, txType) / 100000000,
```
Comment (lsheva) [inline src/utils/createTransactionParser.js:141]: you have 10^8 value here
please use helper function to give some idea about what we converting to what
like `function weiToEther(wei: number): number`

File: src/reducers/wallet.js (hunk 1 above)
Comment (srt0422) [reply src/reducers/wallet.js:27]: yes please

Comment (alex-sandrk) [reply src/reducers/wallet.js]: updated

File: src/utils/createTransactionParser.js (hunk 2 above)
Comment (alex-sandrk) [reply src/utils/createTransactionParser.js]: updated

**Pull request merged**